sample/httpx.py

```diff
@@ -0,0 +1,49 @@
+import httpx
+
+from .oauth2cli import HttpClient, Response
+
+
+class DefaultHttpClient(HttpClient):
+    """
+    Default HTTP Client
+    """
+    def __init__(self):
+        """
+        Constructor for the DefaultHttpClient
+
+        verify=True,  # type: Union[str, True, False, None]
+        proxies=None,  # type: Optional[dict]
+        """
+        self.session = httpx.Client()
+
+    def post(self, url, params=None, data=None, headers=None, **kwargs):
+        if params is None:
+            params = {}
+        params.update(**kwargs)
```

Comment:
Is this how "httpx" handle `kwargs`? Have we run e2e test cases with this setup?

Reply:
Haha, this was supposed to not be checked in but I dint realize pycharm adding it in by default, this is just me trying to use httpx, I'll remove these files in the next commit